src/wrapper.ts

```diff
@@ -52,15 +43,27 @@ const ignoreConfig = [
   /169\.254\.\d+\.\d+.*/gm,
 ];
 
+const lumigoInstrumentationList: Instrumentor[] = [
+  new LumigoExpressInstrumentation(),
+  new LumigoHttpInstrumentation(),
+];
+
+const instrumentationList: InstrumentationBase[] = lumigoInstrumentationList.map((i) =>
+  i instanceof LumigoHttpInstrumentation
```

Comment:
perhaps its better to move this ignore conf inside the HTTP instrumentor constructor to avoid this specific "?" operator